### PR TITLE
#40 Added screen "About"

### PR DESCRIPTION
**Pull request by Chopikov**

### CODE REVIEW

File: app/src/main/res/layout/activity_about.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        android:text="@string/msg_description"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/image_view_logo"
+        app:layout_constraintVertical_bias="0.22" />
+
+    <include
+        android:id="@+id/toolbar_start"
```
Comment (maik-nack) [inline app/src/main/res/layout/activity_about.xml]: Please, change id to `toolbar_about`

File: app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                onBackPressed();
+                return true;
+            default:
+                return super.onOptionsItemSelected(item);
+        }
+    }
+
+    public void onBackPressed() {
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java:51]: Please, make this method with `@Override`

File: app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        finish();
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java:53]: Please, add `super.onBackPressed();` before it

File: app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        overridePendingTransition(R.anim.slide_in_right, R.anim.slide_out_right);
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java]: Please, wrap this line

File: app/src/main/java/com/mithridat/nonoconverter/ui/start/StartActivity.java [hunk 5]
```diff
@@ -47,10 +48,10 @@ public void onClick(View v) {
                         R.anim.slide_out_left);
                 break;
             case R.id.button_about:
-                Toast.makeText(this,
-                        "Open 'about' screen",
-                        Toast.LENGTH_LONG)
-                        .show();
+                Intent intentAbout = new Intent(this, AboutActivity.class);
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/start/StartActivity.java]: Use variable `intent` or delete it

File: app/src/main/res/layout/activity_about.xml [hunk 6]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorLightBlue"
+    tools:context=".ui.start.StartActivity">
```
Comment (maik-nack) [inline app/src/main/res/layout/activity_about.xml]: Change context to `.ui.about.AboutActivity`

File: app/src/main/res/layout/activity_about.xml [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+        android:layout_height="157dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/linear_layout_about"
+        app:layout_constraintVertical_bias="0.15"
+        app:srcCompat="@drawable/panda_thumbnail" />
+
+    <LinearLayout
```
Comment (maik-nack) [inline app/src/main/res/layout/activity_about.xml:17]: It looks like optional layout

File: app/src/main/res/layout/activity_about.xml (hunk 1 above)
Comment (maik-nack) [reply app/src/main/res/layout/activity_about.xml]: You changed the id only in the layout file but forgot in the java class.

**Comment by maik-nack** [comment]
It may be better if you use the ScrollView in the layout file

File: app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+import androidx.appcompat.widget.Toolbar;
+
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.mithridat.nonoconverter.R;
+
+/**
+ * Class for the launcher activity. This is the first thing users see of our app.
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/about/AboutActivity.java]: Please, change comment

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Review by maik-nack — APPROVED** [review]
Looks great!

**Pull request merged**